tests/ui/test/specs/organisations-search.spec.ts

```diff
@@ -30,7 +30,7 @@ test.describe('As a user I want to be able to search for an organisation', {
 
   test('The organisation search page is presented correctly', async () => {
     await test.step('The Organisation search label is visible', async () => {
-      expect(orgSearchPage.getSearchLabel()).toBeVisible;
+      await expect(orgSearchPage.getSearchLabel()).toBeVisible;
```

Comment:
After investigating this might be a wider spread issue that would increase the scope of the ticket - if agreed I would be keen to have a ticket for this instead